app/services/apply_api/import_application.rb

```diff
@@ -37,5 +38,9 @@ def provider_code
     def apply_id
       @apply_id ||= application_data["id"]
     end
+
+    def provider_a_hei?
+      application_data["attributes"]["course"]["training_provider_type"] == "university"
```

Comment:
Might be good to be a bit defensive here. If one of these keys is missing from the payload the job that calls this is going to retry and spam us with errors. There's no point retrying if the payload we receive isn't as expected. Maybe we could catch a `NoMethodError` here and raise something that we could catch in the job and not retry?